### PR TITLE
display profiles in column for static apps

### PR DESCRIPTION
**Pull request by matteo-png**
related to https://github.com/jhipster/jhipster-control-center/issues/136

### CODE REVIEW

**Comment by matteo-png** [comment]
Result of instance view with profile and status.

JCC Static (profile,status)

**Comment by matteo-png** [comment]
Result of instance view with all for the static apps.
Capture d’écran 2021-06-08 101520

**Comment by matteo-png** [comment]
Host with port in column Instance
Capture d’écran 2021-06-08 150624

File: src/main/webapp/app/applications/instance/instance.component.ts [hunk 1]
```diff
@@ -97,6 +149,47 @@ export default class JhiInstance extends Vue {
       .catch(error => console.warn(error));
   }
 
+  public getBadgeClass(statusState: any): string {
+    if (statusState === 'UP') {
```
Comment (qmonmert) [inline src/main/webapp/app/applications/instance/instance.component.ts:153]: I'll do
```
if (statusState === 'UP') {
  return 'badge-success';
}
return 'badge-danger';
```

File: src/main/webapp/app/applications/instance/instance.vue [hunk 2]
```diff
@@ -23,20 +24,19 @@
...
               <span class="badge badge-primary">{{ instance.metadata.version }}</span>
             </td>
             <td class="table-hover">
-              <span v-if="instance.metadata.hasOwnProperty('git-commit') && instance.metadata['git-commit']" class="badge badge-dark">
-                {{ instance.metadata['git-commit'] }}
-              </span>
-              <span v-if="instance.metadata.hasOwnProperty('git-branch') && instance.metadata['git-branch']" class="badge badge-dark">
-                {{ instance.metadata['git-branch'] }}
-              </span>
+              <span class="badge badge-dark"> {{ versionInstance(instance) }}</span>
```
Comment (qmonmert) [inline src/main/webapp/app/applications/instance/instance.vue:36]:
``` diff
+ <span class="badge badge-dark">{{ versionInstance(instance) }}</span>
- <span class="badge badge-dark"> {{ versionInstance(instance) }}</span>
```

**Review by chrisdns — APPROVED** [review]

File: src/main/webapp/app/applications/instance/instance.component.ts [hunk 3]
```diff
@@ -56,12 +62,58 @@ export default class JhiInstance extends Vue {
       .findAllGatewayRoute()
       .then(res => {
```
[the remark this answers is not in the document]
Comment (avdev4j) [reply src/main/webapp/app/applications/instance/instance.component.ts:64]: not sure to understand the point here.

**Pull request merged**